experimental/packages/opentelemetry-api-logs/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@opentelemetry/api-logs",
+  "version": "0.0.1",
```

Comment:
I think the version should be the same for all experimental packages. but maybe it's updated automatically during release anyway.